lib/interface.ts

```diff
@@ -14,3 +14,14 @@ export interface IOurResponse extends restify.Response {
     sendEnd?: (status: any, message: string) => Promise<void>;
     sendError?: (err: any, where: string, reason?: any) => Promise<void>;
 }
+
+export interface ISocket {
+    log: bunyan.Logger; // Add bunyan logger to socket
+    blpSession: BAPI.Session;  // Add blpSession to socket
+    isConnected(): boolean;
+    getIP(): string;
+    getCert(): any;
+    emit(name: string, ...args: any[]): void;
+    disconnect(): void;
+    on(event: string, listener: Function): void;
```

Comment:
Well, doing so will force the wrapper implement all the `EventEmitter` functions, which I am not sure if it is worth it. I only use `on` & `emit` though.